RestApi/Sepes.RestApi/Services/PodService.cs

```diff
@@ -11,6 +11,7 @@ namespace Sepes.RestApi.Services
     public interface IPodService
     {
         Task Set(Pod newPod, Pod based, IEnumerable<User> newUsers, IEnumerable<User> basedUsers);
+        Task Delete(Pod pod);
```

Comment:
If we follow the architecture the same rules as with study service.
Set(null, pod) = delete
Set(pod, pod) = update
Set(pod, null) = create

Reply:
This has been done in https://github.com/equinor/sepes/pull/151/commits/1dbaabbb56a67d2ed9e404561b1a3a2d2f73652c